code/modules/client/preference_setup/loadout/loadout_xeno/unathi.dm

```diff
@@ -98,10 +98,12 @@
 
 /datum/gear/eyes/wasteland_goggles
 	display_name = "wasteland goggles"
-	path = /obj/item/clothing/glasses/safety/goggles/wasteland
+	path = /obj/item/clothing/glasses/welding/wasteland
 	whitelisted = list(SPECIES_UNATHI)
 	origin_restriction = list(/decl/origin_item/origin/wastelander)
 	sort_category = "Xenowear - Unathi"
+	allowed_roles = list("Chief Engineer", "Engineer", "Atmospheric Technician", "Engineering Apprentice", "Machinist")
+	cost = 2
```

Comment:
this is kinda costly for something you can easily find ingame with equivalent access.

Reply:
I'd be fine lowering it - I just assumed something with mechanical impact should be upped.